Example/KlarnaOnDemandExample/MainViewController.m

```diff
@@ -15,6 +15,7 @@ - (void)viewDidLoad {
   [super viewDidLoad];
 
   [self initializeUIElements];
+  [self updateUIElements];
```

Comment:
I called initializeUIElements on viewDidLoad, and when I received 'userReady' event, I just set registrationButton.hidden=true and preferencesButton.hidden=false.